.editorconfig

```diff
@@ -0,0 +1,5 @@
+[*]
+insert_final_newline = true
+
+[*.gradle]
+indent_size = 2
```

Comment:
https://square.github.io/leakcanary/dev-env/ suggests to import Square's codestyle setup, but looking at the project it is not respected, i.e. `/build.gradle` has different indentation than `/shark/build.gradle`
I though I can suggest to add `.editorconfig` file which would modify default project settings.